index.html

```diff
@@ -1501,6 +1501,52 @@ <h4>LinkedDomains</h4>
 
   </section>
 
+  <section>
+    <h1>Representations</h1>
+    <p>This table provides a reference for media types and the associated specifications for producing and consuming those representations.</p>
+    <table class="simple" style="width: 100%;">
+      <thead>
+        <tr>
+          <th>Media Type</th>
+          <th>Specification</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr id="application-did-json">
+          <td>application/did+json</td>
+          <td>
+            <a href="https://www.w3.org/TR/did-core#json">DID Core</a>
+          </td>
+        </tr>
+        <tr id="application-did-ld-json">
+          <td>application/did+ld+json</td>
+          <td>
+            <a href="https://www.w3.org/TR/did-core#json-ld">DID Core</a>
+          </td>
+        </tr>
+        <tr id="application-did-cbor">
+          <td>application/did+cbor</td>
+          <td>
+            <a href="https://www.w3.org/TR/did-core#cbor">DID Core</a>
+          </td>
+        </tr>
+        <tr id="application-did-dag-cbor">
+          <td>application/did+dag+cbor</td>
+          <td>
+            <p class="issue" data-number="252"></p>
+          </td>
+        </tr>
+        <tr id="application-did-yaml">
+          <td>application/did+yaml</td>
+          <td>
+            <p class="issue" data-number="253"></p>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
```

Comment:
I agree with https://github.com/w3c/did-spec-registries/pull/254/files#r576489864.

You could simply move the `<p>` pointers to https://github.com/w3c/did-spec-registries/issues/252 and https://github.com/w3c/did-spec-registries/issues/253 to this spot here below the table. This way they would still appear in the document without actually "being registered" yet.

Reply:
I merged just 'cause people are complaining... can we clean this up in a separate PR. @peacekeeper, could you please raise a PR/issue or maybe @OR13?